src/sdk/pynni/nni/compression/torch/builtin_pruners.py

```diff
@@ -320,61 +215,65 @@ def calc_mask(self, layer, config):
         weight = layer.module.weight.data
         op_name = layer.name
         op_type = layer.type
-        assert op_type == 'Conv2d', 'L1FilterPruner only supports 2d convolution layer pruning'
+        assert op_type == 'BatchNorm2d', 'SlimPruner only supports 2d batch normalization layer pruning'
         if op_name in self.mask_calculated_ops:
             assert op_name in self.mask_dict
             return self.mask_dict.get(op_name)
         mask = torch.ones(weight.size()).type_as(weight)
         try:
```

Comment:
It's unnecessary  to use 'try finally' here.

Reply:
resolved